src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Extensions/StartupExtensions.cs

```diff
@@ -94,11 +94,6 @@ public static IServiceCollection AddExternalLoginProviders(this IServiceCollecti
 
         public static void UpdateDatabase(this IApplicationBuilder app)
         {
-            if (app == null)
```

Comment:
please revert null check here, but remove 
if(app != null)
because we are sure that it is already not null